reference/open223-resources.md

```diff
@@ -6,34 +6,68 @@ The [Open223 GitHub organization](https://github.com/open223) was created in con
 ## [Open223 Models](https://models.open223.info/)
 
 This site contains the canonical sources for example models.
+Each page listed in the table of contents (on the left-hand side of the page) describes an example building or system modeled in 223.
+On each page you will find the following:
+- short description of what is being modeled
+- supplementary materials and/or diagrams, where applicable
+- download links for the model (both Turtle and JSON-LD are provided)
+- links to example queries against the model; queries are hosted on the [Open223 query page](https://query.open223.info)
+- a table of entities found inside the model
 
 ## [Open223 Explore](https://explore.open223.info/)
 
-This page hosts autogenerated documentation of the 223P ontology, emphasizing the availability and correct use of concepts and properties defined by the ontology.
+This page hosts autogenerated documentation of the 223 ontology, emphasizing the availability and correct use of concepts and properties defined by the ontology.
+The Explore tool divides the ontology up across four tabs.
 
-### Navigation
+![](/_static/images/explore-tabs.png)
 
-This section will describe how to navigate the site, which allows one to traverse the standard. It uses a hierarchical structure based on classes within the standard for navigation. This section will describe how to use this page to explore standard 223P.
+#### Classes
+The classes tab organizes the classes of the standard based on their class hierarchy.
+One can click on the class name to view a description of the class and its constraints, or click on the "+" to the left of the class name to view all the subclasses of that class.
+For example, all 223 classes are a subclass of `s223:Concept`, and can be accessed by clicking the "+" to the left of "Concept".
 
-This section is organized based on the tabs on the left hand side of the explore page. It will describe how to use this website to explore standard 223P.
+![](/_static/images/explore-classes-domainspace.png)
 
-Insert image of left hand tab
+The **Definition** section describes what relations *instances* of the class must have.
+The **Properties** section below lists all of the relation constraints the class as inherited, including: which of those relations are optional (marked with a `?`), can happen 0 or more times (marked with a `*`), must happen
+at least once (marked with a `+`), or are necessary (no additional markings, like in the screenshot above).
+Additionally all links highlighted in yellow on a given class page can be used to navigate to a description of that class or property.
 
-#### Classes
+The definitions listed on the Explore tool are *not* definitive but instead capture common usage; refer to the standard for the full and complete definition.
```

Comment:
We *might* include a copy of the Standard's Definitions section. Update later if so.